client/trino-client/src/test/java/io/trino/client/auth/external/TestExternalAuthenticator.java

```diff
@@ -52,6 +52,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please add description to the commit message why it fixes the flaky test.

Reply:
A couple of typos in the commit message.